From 68480dab82c60c12eb7bbdb31fb71fdacc7fe709 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 19 May 2016 21:45:29 +0900
Subject: [PATCH 131/909] clk: renesas: r8a7795: Add VCP4 clock

commit 05dbabf6d386e5c479194282a9c7ff7b384ce8f9 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds VCPLF, VCPL4, VDPB clocks to the R8A7795 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a7795-cpg-mssr.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/clk/renesas/r8a7795-cpg-mssr.c b/drivers/clk/renesas/r8a7795-cpg-mssr.c
index 1fa4425be8af..6af729c46d80 100644
--- a/drivers/clk/renesas/r8a7795-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7795-cpg-mssr.c
@@ -124,6 +124,9 @@ static struct mssr_mod_clk r8a7795_mod_clks[] __initdata = {
 	DEF_MOD("fdp1-2",		 117,	R8A7795_CLK_S2D1), /* ES1.x */
 	DEF_MOD("fdp1-1",		 118,	R8A7795_CLK_S0D1),
 	DEF_MOD("fdp1-0",		 119,	R8A7795_CLK_S0D1),
+	DEF_MOD("vcpl4",		 129,	R8A7795_CLK_S2D1), /* ES1.x */
+	DEF_MOD("vcplf",		 130,	R8A7795_CLK_S2D1),
+	DEF_MOD("vdpb",			 131,	R8A7795_CLK_S2D1),
 	DEF_MOD("scif5",		 202,	R8A7795_CLK_S3D4),
 	DEF_MOD("scif4",		 203,	R8A7795_CLK_S3D4),
 	DEF_MOD("scif3",		 204,	R8A7795_CLK_S3D4),
@@ -401,6 +404,7 @@ static const struct mssr_mod_reparent r8a7795es1_mod_reparent[] __initconst = {
 
 static const unsigned int r8a7795es2_mod_nullify[] __initconst = {
 	MOD_CLK_ID(117),			/* FDP1-2 */
+	MOD_CLK_ID(129),			/* VCPL4 */
 	MOD_CLK_ID(327),			/* USB3-IF1 */
 	MOD_CLK_ID(600),			/* FCPVD3 */
 	MOD_CLK_ID(609),			/* FCPVI2 */
-- 
2.17.1

